tests/Eccube/Tests/Web/Admin/IndexControllerTest.php

```
@@ -193,4 +197,119 @@ protected function createChangePasswordFormData()
+     */
+    public function testDashboardPerformanceWithLargeDataset()
+    {
+        // Clear existing orders to ensure test isolation
+        $this->deleteAllRows(['dtb_order']);
+
+        $Customer = $this->createCustomer();
+        $OrderNew = $this->orderStatusRepository->find(OrderStatus::NEW);
+
+        // Create 100 orders to simulate realistic load
+        $orderCount = 100;
+        $startTime = microtime(true);
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$startTime'. (undefined)

(UnusedLocalVariable)